core/math/math_funcs.h

```diff
@@ -33,14 +33,14 @@
 
 #include "core/error/error_macros.h"
 #include "core/math/math_defs.h"
-#include "core/math/random_pcg.h"
 #include "core/typedefs.h"
 
 #include "thirdparty/misc/pcg.h"
 
-#include <float.h>
```

Comment:
So as you can see this change won't work, and is the major change here moving `float.h` out of this file, but that won't compile on windows, which is a good indication of how these kinds of changes have potential fallout and needs to be done carefully